config/v201/component_config/standardized/SecurityCtrlr.json

```diff
@@ -177,6 +177,38 @@
       "maximum": 3,
       "default": "1",
       "type": "integer"
+    },
+    "AllowCSMSRootCertificateInstallWhenLowSecurityProfile": {
+      "variable_name": "AllowCSMSRootCertificateInstallWhenLowSecurityProfile",
```

Comment:
"Low" security profile leaves room for interpretation, it could be a bit more explicit still ;)

Reply:
it is now `AllowCSMSRootCertInstallWhenSecurityProfile1`